### PR TITLE
Nits

### PR DESCRIPTION
**Pull request by mpharrigan**
found during my travels

### CODE REVIEW

File: qualtran/_infra/composite_bloq.py [hunk 1]
```diff
@@ -771,7 +771,9 @@ def add_register(
         return None
 
     @classmethod
-    def from_signature(cls, signature: Signature, add_registers_allowed=False):
+    def from_signature(
```
Comment (fdmalone) [inline qualtran/_infra/composite_bloq.py:774]: does mypy / pylint not raise an error without a type hint?

File: qualtran/_infra/composite_bloq.py [hunk 2]
```diff
@@ -771,7 +771,9 @@ def add_register(
...
+        cls, signature: Signature, add_registers_allowed=False
```
Comment (fdmalone) [inline qualtran/_infra/composite_bloq.py:775]: nit on your nits: add_registers_allowed is missing a type hint

**Review by fdmalone — CHANGES_REQUESTED** [review]

File: qualtran/_infra/composite_bloq.py (hunk 1 above)
Comment (mpharrigan) [reply qualtran/_infra/composite_bloq.py:774]: it does not

File: qualtran/_infra/composite_bloq.py (hunk 2 above)
Comment (mpharrigan) [reply qualtran/_infra/composite_bloq.py]: done

**Review by fdmalone — APPROVED** [review]

**Pull request merged**